src/app.ts

```diff
@@ -340,6 +340,38 @@ if (config.IDP === "selfcare") {
       res.end();
     }
   );
+
+  app.use(
+    "/organizations/:organizationFiscalCode/services/*",
+    sessionTokenVerifier,
+    async (req, res) => {
+      const url = `${config.ORGANIZATION_SERVICES_URL}/organizations/${req.user?.organization.fiscal_code}/services${req.params[0]}`;
```

Comment:
What if `user` is null?

Reply:
More: are you sure `user` always has a related `organization`? What if we are in a DevPortal installation (which would be the case  for admin users)?

Reply:
> What if `user` is null?

Organization Fiscal Code needs to be a value in the path "/organizations/:organizationFiscalCode/services/*", so I'm wondering if we can read the parameter in the URL path in this manner:

```
app.use(
    "/organizations/:organizationFiscalCode/services/*",
...
const organizationFiscalCode = req.params["organizationFiscalCode"];
const url = `${config.ORGANIZATION_SERVICES_URL}/organizations/${organizationFiscalCode}/services/${req.params[0]}`;
```